community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/load_bq.py

```diff
@@ -282,6 +284,26 @@ def bq_submit(jobs):
     print(f"successfully loaded {len(jobs)} jobs")
 
 
+def batched_bq_submit(
+    jobs, submit_function=bq_submit, bq_row_batch_size=BQ_ROW_BATCH_SIZE
+):
+    """Submit sacct data in batches of size bq_row_batch_size
+
+    Args:
+        jobs: A list of dictionaries of sacct accounting data.
+        submit_function: The method to submit the jobs to BigQuery with. Defaults to bq_submit.
+        bq_row_batch_size: The accounting data will be submitted to BigQuery in
+            batches of this size.
+    """
+    num_batches = int(math.ceil(len(jobs) / bq_row_batch_size))
```

Comment:
nit. `math.ceil` already returns `int`, no need to cast.

Reply:
I would hold off on review for a minute. going to make a bigger change.

Reply:
done.